cypress/e2e/pages/apply/household/add-person.cy.ts

```diff
@@ -45,8 +44,7 @@ describe('Add person to household ', () => {
     HouseholdPage.visit();
     HouseholdPage.getHouseholdPage().should('be.visible');
 
-    AddPersonPage.visit();
-    AddPersonPage.getAddPersonPage().should('be.visible');
```

Comment:
I guess this could be left as a check after clicking the add person button to ensure we are on the correct page